migrations/v20x/remove_rewrite_conf.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ *
+ * phpBB Directory extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2017 ErnadoO <http://www.phpbb-services.com>
```

Comment:
Maybe it would be better to create a dedicated commit.
https://github.com/ErnadoO/ext-phpbb-directory/search?utf8=✓&q=http%3A%2F%2Fwww.phpbb-services.com&type=Code :stuck_out_tongue_closed_eyes: